### ISSUE TITLE
Enhancement Expansion of support for WorldGenerators

### ISSUE THREAD

**Issue by Dev7ex** [Dev7ex/MultiWorld#31]
The WorldGenerators should support the following things in the future.

- WorldGenerator files. This means that the user should be able to create a script in which he can decide for himself how what is generated.

- WorldGenerator with single biome support

- Ability to assign arguments to WorldGenerators already known from the CleanroomGenerator plugin

**Comment by ElTuna** [comment]
yes please, I'm using the command `world import spawn CUSTOM -g CleanroomGenerator:.` and it says the generator doesn't exist 😭
